### PR TITLE
Update ElementwiseOpToLLVM.cpp

### PR DESCRIPTION
**Pull request by etiotto**

### CODE REVIEW

**Comment by etiotto** [comment]
Depends on #2973

File: third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp [hunk 1]
```diff
@@ -860,73 +836,10 @@ inline Type getElementType(Value value) {
...
-    outValues.push_back(bitcast(vec, i32_ty));
-  }
-  return outValues;
-}
-
 typedef std::function<SmallVector<Value>(Location, ConversionPatternRewriter &,
                                          const SmallVector<Value> &)>
     ConverterT;
 
-class MultipleOperandsRange
```
Comment (etiotto) [inline third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp:915]: Same class exist in common header file, so we can remove it here.

File: third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp [hunk 2]
```diff
@@ -860,73 +836,10 @@ inline Type getElementType(Value value) {
   return type;
 }
 
-inline SmallVector<Value> unpackI32(const SmallVector<Value> &inValues,
```
Comment (etiotto) [inline third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp:863]: Unused

File: third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp [hunk 3]
```diff
@@ -860,73 +836,10 @@ inline Type getElementType(Value value) {
...
-    auto vecType = vec_ty(eltType, 32 / eltType.getIntOrFloatBitWidth());
-    auto vec = bitcast(v, vecType);
-    for (int i = 0; i < 32 / eltType.getIntOrFloatBitWidth(); i++) {
-      outValues.push_back(extract_element(vec, i32_val(i)));
-    }
-  }
-  return outValues;
-}
-
-inline SmallVector<Value> packI32(const SmallVector<Value> &inValues,
```
Comment (etiotto) [inline third_party/intel/lib/TritonIntelGPUToLLVM/ElementwiseOpToLLVM.cpp:887]: Unused

**Review by whitneywhtsang — APPROVED** [review]

**Pull request merged**